cli/tests/concurrent_test.go

```diff
@@ -30,6 +30,13 @@ func TestConcurrent(t *testing.T) {
 		t.Skip("Skipping concurrent tests")
 	}
 
+	// skipping concurrent tests for now
```

Comment:
Maybe it's better to just do a t.Skip("reason why") so we get it logged when running the tests